internal/distro/rhel8/pipelines.go

```diff
@@ -698,7 +698,13 @@ func osPipeline(t *imageType,
 			},
 		}))
 	}
-
+	greenbootConfig := &osbuild.GreenbootConfig{
+		MonitorServices: []string{"sshd", "NetworkManager"}, //sshd,NetworkManager are default services to be monitored by greenboot
+	}
+	if newGreenbootConfig := c.GetGreenbootConfig(); newGreenbootConfig != nil {
+		greenbootConfig.MonitorServices = newGreenbootConfig.MonitorServices
+	}
+	p.AddStage(osbuild.NewGreenbootConfig(greenbootConfig))
```

Comment:
This is part of the `osPipeline` which handles (almost) all images, not just ostree, so this code would add a greenboot configuration to everything.
You should move it to the `if t.rpmOstree {}` block above to make it ostree only.

Reply:
Done